### ISSUE TITLE
Need an Amazon Fire to be able to play osmo genius starter kit, coder, creative kit.

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#843]
Tech for https://github.com/kinu-garage/30y-130s_life/issues/1695

**Comment by 130s** [comment]
- Device model: Fire HD 8 (7th gen)
- Serial: (Hidden)
- Fire OS version (2023/06/10): 5.7.0.0 (installed on 2022/06/25)
- Setup instruction for Fire: https://assets.playosmo.com/2021/onboarding/osmo-fire-setup-guide.pdf
   - Says this.
      > Osmo has limited compatibility with the following Fire Tablets (which cannot be upgraded to Fire OS 7.0):
      > - Amazon Fire 7 (7th Gen, 2017 Release)
      > - Amazon Fire HD 8 (7th Gen, 2017 Release)
      > The tablets listed above are no longer being supported by Amazon. If you have previously downloaded Osmo apps on one of these devices, they should continue to work. However, they will not receive new updates or content and if you delete the app, you will not be able to download it again. As a result, we are limited in our ability to provide technical support for these models.
- "Parent" account on this device password: (hidden)

[Setup instruction on Fire (playosmo.com)](https://assets.playosmo.com/2021/onboarding/osmo-fire-setup-guide.pdf).

Stuck at step10:
- Problem-1. I do not see AK's account in the target.
- Problem-2. No apps prefixed by Osmo shows up on the sending sources even though on "parent account" I see them on the device.

AK's account is "hidden" on black Fire for some reason. Trying https://www.amazonforum.com/s/question/0D54P000084zzUsSAI/why-is-my-childs-profile-hidden but then blocked by the Amazon Prime account's password, which the one who set up (not me) can't recall.

Decided to factory reset (ref. https://www.asurion.com/connect/tech-tips/how-to-reset-amazon-fire-tablet/), which requires 30% battery remaining that it doesn't have right now. Charging now.

**Comment by 130s** [comment]
As in https://github.com/kinu-garage/30y-130s_life/issues/1695#issuecomment-1676597526, this is no longer needed.
